test/developmentAccount.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { expect } from 'chai'
+
+import Metrics from '../src/core'
+import { UnknownEntityError } from '../src/error'
+import { DevelopmentAccount, DevelopmentAccountSorting } from '../src/models/developmentAccount'
+import { User } from '../src/models/user'
+import { createNewUserSession, getMetricsInstance } from './utils/fixtures'
+
+describe('Development Account', function () {
+  let metricsInstance: Metrics
+  let user: User
+  let createdDevelopmentAccount: DevelopmentAccount
+
+  before(async function () {
+    metricsInstance = getMetricsInstance()
+    const userSession = await createNewUserSession(metricsInstance)
+    user = userSession.user
+  })
+
+  after(async function () {
+    await user.delete()
```

Comment:
Need to delete the `createdDevelopmentAccount` once finished.

Reply:
`createdDevelopmentAccount` is deleted in the last test in this spec